extensions/notification/utils/config.js

```diff
@@ -32,11 +32,13 @@ function configArrayToMap(bnConfigs) {
 function validateEntryWithFilter(filterRules, entry) {
     const { key } = entry;
     return filterRules.every(rule => {
-        if (rule.name === constants.nameFilter.prefix) {
-            return key.startsWith(rule.value);
+        const { name, value } = rule;
+        const { prefix, suffix } = constants.nameFilter;
+        if (name.toLowerCase() === prefix.toLowerCase()) {
```

Comment:
Is this correct if both a prefix and a suffix are given? Meaning the fact that we return from the function with the first check's result.

Reply:
`filterRules` is an array with either `prefix` or `suffix` and the it is checked for all the rules. `every` is used on the array.